templates/vue/src/components/NodeModal.vue

```diff
@@ -334,6 +334,106 @@
         >
           <quiz-modal :node="node" />
         </b-tab>
+        <b-tab
+          v-if="node.tydeType === tydeTypes.MODULE"
+          title="Spaceship Icons"
+        >
+          <div id="modal-spaceship-icons">
+            <b-form-group label="Planet View Icon - not earned">
+              <file-upload
+                v-model="node.typeData.planetViewNotEarnedIconUrl"
+                placeholder="Enter link (starting with http)"
+              />
+            </b-form-group>
+            <b-form-group label="Planet View Icon - earned">
+              <file-upload
+                v-model="node.typeData.planetViewEarnedIconUrl"
+                placeholder="Enter link (starting with http)"
+              />
+            </b-form-group>
+            <b-form-group label="Spaceship Part - not earned">
```

Comment:
Please add headings to group items belonging to "Planet View Icon" and "Spaceship Cockpit Image" separately. See the Appearance tab to see how the headings are implemented and replicate that here.